src/service/faucet/faucet.ts

```diff
@@ -15,13 +16,20 @@ import { TxIdResponse } from './faucet.dto'
 export class Faucet {
   private readonly connector: TatumConnector
   private readonly config: TatumConfig
+  private readonly logger: Logger
 
   constructor(private readonly id: string) {
     this.connector = Container.of(this.id).get(TatumConnector)
     this.config = Container.of(this.id).get(CONFIG)
+    this.logger = Container.of(this.id).get(LOGGER)
   }
 
   async fund(address: string): Promise<ResponseDto<TxIdResponse>> {
+    if (!this.config.apiKey?.v4)
```

Comment:
Even though it is technically a single line of code, not a full block, I think since it is extended to multiple rows it would look better to add `{}` brackets, but just a minor nitpick.